docker-compose.yaml

```diff
@@ -1,17 +1,17 @@
 #https://docs.docker.com/registry/deploying/#deploy-your-registry-using-a-compose-file
 services:
   nginx:
-    image: "nginx:stable-alpine"
+    image: 'nginx:stable-alpine'
     ports:
       - 443:443
     restart: unless-stopped
     volumes:
       - ./config/etc/nginx/nginx.conf:/etc/nginx/nginx.conf
       - /etc/letsencrypt:/etc/letsencrypt
-    depends_on: 
+    depends_on:
```

Comment:
We'll need to add `docker-auth` here too, so nginx waits on it to exist.

Reply:
Added